src/ScottPlot/Plot/Plot.Render.cs

```diff
@@ -172,10 +183,14 @@ public Bitmap RenderLegend(bool lowQuality = false)
         /// Save the plot as an image
         /// </summary>
         /// <param name="filePath">file path for the images (existing files will be overwritten)</param>
+        /// <param name="bitmapWidth">the width (in pixels) of the output bitmap</param>
```

Comment:
`bitmapWidth` may not be the best parameter name as this method is not restricted to only saving to a bitmap image (or even bitmap-like formats such as PNG).

`imageWidth` or `outputWidth` probably makes the most sense.